### ISSUE TITLE
Spotify just updated and the scripts stopped working

### ISSUE THREAD

**Issue by justin808** [phpfunk/alfred-spotify-controls#35]
Latest build of Spotify and this no longer works:

```
on alfred_script(q)
    tell application "Spotify"
        playpause
    end tell
end alfred_script
```

Anybody know how to troubleshoot?

![screenshot_3_14_15__9_11_pm](https://cloud.githubusercontent.com/assets/1118459/6655034/a6b772ae-ca8e-11e4-8e39-6ca1882c0f74.jpg)

**Comment by matbrady** [comment]
Solution src: [community.spotify.com](https://community.spotify.com/t5/Help-Desktop-Linux-Mac-and/Apple-scripting-broken-in-1-0-1-988-g8f17a348/td-p/1029434)

> Just right click spotify.app, select show contents, and use any text editor to edit the `Info.plist` file

```
// old line (towards the bottom)
<string>Spotify.sdef</string>
// new line
<string>applescript/Spotify.sdef</string>
```

Then restart spofity and Alfred 2

**Comment by bradical** [comment]
@matbrady did this work for you in the most recent release `1.0.2.6.g9977a14b`? It worked for me in the previous version, but [didn't work with this one](https://github.com/phpfunk/alfred-spotify-controls/issues/34#issuecomment-81708960).

**Comment by justin808** [comment]
You definitely have to quick and restart both spotify and alfred. No computer restart is needed.

**Comment by bradical** [comment]
Nice. Restarting Alfred did the trick. I restarted Spotify multiple times to no avail but didn't realize Alfred restart was necessary. Thanks!

**Comment by justin808** [comment]
@matbrady This just broke again:

https://community.spotify.com/t5/Live-Ideas/Bring-back-Applescript-support/idc-p/1068934#M31916

I'm guessing Spotify updated again, and re-doing the fix to the plist file and restarting got this working again.

**Comment by matbrady** [comment]
@justin808 yeah I noticed. It's unfortunate we have to patch this every time they push an update that doesn't enable AppleScripts.

**Comment by bradical** [comment]
Looks like they'll be patching it in a future desktop version: https://community.spotify.com/t5/Live-Ideas/Bring-back-Applescript-support/idc-p/1068934#M31916
